library/Fission/User/CID/Mutation.hs

```diff
@@ -55,3 +57,16 @@ createX userId hashes = runDBNow \now -> do
 
   insertMany_ toInsert
   return newHashes
+
+destroyAll :: MonadDB m => [Key UserCid] -> Transaction m ()
+destroyAll userCidIds =
+  delete <| from \userCid -> where_ (userCid ^. UserCidId `in_` valList userCidIds)
+
+destroyExact :: MonadDB m => UserId -> CID -> Transaction m ()
+destroyExact userId cid = do
+    delete <| from \userCid ->
+      where_ (selectExact userCid)
+  where
+    selectExact userCid =
+          userCid ^. UserCidCid    ==. val cid
+      &&. userCid ^. UserCidUserFk ==. val userId
```

Comment:
We've been following 2-space indentation, but this is another popular style.

```haskell
destroyExact :: MonadDB m => UserId -> CID -> Transaction m ()
destroyExact userId cid = do
  delete <| from \userCid ->
    where_ (selectExact userCid)
  where
    selectExact userCid =
          userCid ^. UserCidCid    ==. val cid
      &&. userCid ^. UserCidUserFk ==. val userId
```

I find that syntax highlighting and whitespace make it clear enough for me. How does it feel to you? Mostly thinking in terms of a longer-term company style, and totally open to opinions.